<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>656</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="5">
       <widget class="QPushButton" name="clearSearch">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: #3498db;
	border: 1px solid #2980b9;
	 color: #ffffff;
	padding: 5px 10px;
	border-radius: 3px;
}
                      
QPushButton:hover {
background-color: #2184cb;border: 1px solid #1c6da5;
}
                      
QPushButton:pressed {
	background-color: #1a548b;
	border: 1px solid #174172;
}

QPushButton:disabled {
	background-color: #d3d3d3;	
	color: #555555;
	border: 1px solid #a3a3a3;
}</string>
        </property>
        <property name="text">
         <string>Очистить поиск</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="6">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Сортировать столбец:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
    background-color: #0E9252;
    border: 1px solid #2980b9;
    color: #ffffff;
    padding: 5px;
    border-radius: 3px;
}

QComboBox:hover {
    background-color: #42A977; 
    border: 1px solid #1c6da5;
}

QComboBox:disabled {
    background-color: #d3d3d3;    
    color: #555555;
    border: 1px solid #a3a3a3;
}</string>
          </property>
          <item>
           <property name="text">
            <string>Имя</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Фамилия</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>по</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_type_sort">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
    background-color: #0E9252;
    border: 1px solid #2980b9;
    color: #ffffff;
    padding: 5px;
    border-radius: 3px;
}

QComboBox:hover {
    background-color: #42A977; 
    border: 1px solid #1c6da5;
}

QComboBox:disabled {
    background-color: #d3d3d3;    
    color: #555555;
    border: 1px solid #a3a3a3;
}</string>
          </property>
          <item>
           <property name="text">
            <string>Возрастанию (А-Я)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Убыванию (Я-А)</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_sort">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #3498db;
	border: 1px solid #2980b9;
	 color: #ffffff;
	padding: 5px 10px;
	border-radius: 3px;
}
                      
QPushButton:hover {
background-color: #2184cb;border: 1px solid #1c6da5;
}
                      
QPushButton:pressed {
	background-color: #1a548b;
	border: 1px solid #174172;
}

QPushButton:disabled {
	background-color: #d3d3d3;	
	color: #555555;
	border: 1px solid #a3a3a3;
}</string>
          </property>
          <property name="text">
           <string>Отсортировать</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="5" column="9">
       <widget class="QPushButton" name="resetTable">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: #3498db;
	border: 1px solid #2980b9;
	 color: #ffffff;
	padding: 5px 10px;
	border-radius: 3px;
}
                      
QPushButton:hover {
background-color: #2184cb;border: 1px solid #1c6da5;
}
                      
QPushButton:pressed {
	background-color: #1a548b;
	border: 1px solid #174172;
}

QPushButton:disabled {
	background-color: #d3d3d3;	
	color: #555555;
	border: 1px solid #a3a3a3;
}</string>
        </property>
        <property name="text">
         <string>Cбросить
результат</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="searchText">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QPushButton" name="addFilter">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: #3498db;
	border: 1px solid #2980b9;
	 color: #ffffff;
	padding: 5px 10px;
	border-radius: 3px;
}
                      
QPushButton:hover {
background-color: #2184cb;border: 1px solid #1c6da5;
}
                      
QPushButton:pressed {
	background-color: #1a548b;
	border: 1px solid #174172;
}

QPushButton:disabled {
	background-color: #d3d3d3;	
	color: #555555;
	border: 1px solid #a3a3a3;
}</string>
        </property>
        <property name="text">
         <string>Добавить фильтр</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pushButton_search">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: #3498db;
	border: 1px solid #2980b9;
	 color: #ffffff;
	padding: 5px 10px;
	border-radius: 3px;
}
                      
QPushButton:hover {
background-color: #2184cb;border: 1px solid #1c6da5;
}
                      
QPushButton:pressed {
	background-color: #1a548b;
	border: 1px solid #174172;
}

QPushButton:disabled {
	background-color: #d3d3d3;	
	color: #555555;
	border: 1px solid #a3a3a3;
}</string>
        </property>
        <property name="text">
         <string>Поиск</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QPushButton" name="clearFilter">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: #3498db;
	border: 1px solid #2980b9;
	 color: #ffffff;
	padding: 5px 10px;
	border-radius: 3px;
}
                      
QPushButton:hover {
background-color: #2184cb;border: 1px solid #1c6da5;
}
                      
QPushButton:pressed {
	background-color: #1a548b;
	border: 1px solid #174172;
}

QPushButton:disabled {
	background-color: #d3d3d3;	
	color: #555555;
	border: 1px solid #a3a3a3;
}</string>
        </property>
        <property name="text">
         <string>Сброс фильтра</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {
    background-color: #0E9252;
    border: 1px solid #2980b9;
    color: #ffffff;
    padding: 5px;
    border-radius: 3px;
}

QComboBox:hover {
    background-color: #42A977; 
    border: 1px solid #1c6da5;
}

QComboBox:disabled {
    background-color: #d3d3d3;    
    color: #555555;
    border: 1px solid #a3a3a3;
}</string>
        </property>
        <item>
         <property name="text">
          <string>Имя</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Фамилия</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Поиск:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="10">
       <widget class="QTableView" name="tableView">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="checkBox">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Точное
совпадение</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Текущая страница: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentPage">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="maxPage">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="prevButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: #3498db;
	border: 1px solid #2980b9;
	 color: #ffffff;
	padding: 5px 10px;
	border-radius: 3px;
}
                      
QPushButton:hover {
background-color: #2184cb;border: 1px solid #1c6da5;
}
                      
QPushButton:pressed {
	background-color: #1a548b;
	border: 1px solid #174172;
}

QPushButton:disabled {
	background-color: #d3d3d3;	
	color: #555555;
	border: 1px solid #a3a3a3;
}</string>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Перейти к странице №:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="goToPage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="nextButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: #3498db;
	border: 1px solid #2980b9;
	 color: #ffffff;
	padding: 5px 10px;
	border-radius: 3px;
}
                      
QPushButton:hover {
background-color: #2184cb;border: 1px solid #1c6da5;
}
                      
QPushButton:pressed {
	background-color: #1a548b;
	border: 1px solid #174172;
}

QPushButton:disabled {
	background-color: #d3d3d3;	
	color: #555555;
	border: 1px solid #a3a3a3;
}</string>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
